packages/calcite-components/src/components/input/input.scss

```diff
@@ -1,4 +1,48 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-input-corner-radius: defines the border radius of the component.
+ * @prop --calcite-input-text-color: defines the text color of the component.
+ * @prop --calcite-input-border-color: defines the border color of the component.
+ * @prop --calcite-input-background-color: defines the background color of the component.
+ * @prop --calcite-input-button-background-color: defines the background color of a button element in the component.
+ * @prop --calcite-input-button-background-color-hover: defines the background color of a :hover-ed button element in the component.
+ * @prop --calcite-input-button-background-color-active: defines the background color of an :active button element in the component.
+ * @prop --calcite-input-icon-color: defines the color of an icon element in the component.
+ * @prop --calcite-input-icon-color-hover: defines the color of an icon element when it's parent is hovered in the component.
```

Comment:
Can the user set this with calcite-input: hover or is it for a different icon besides the one from ‘icon’ prop?

Reply:
it's different.

Reply:
Maybe `input-button-icon-color` and `input-button-icon-color-hover` just to be clearer? From the name / description it just sounds like the hover color for the icon set above (which can be affected with a parent-level pseudo state override)